ee/clickhouse/materialized_columns/columns.py

```diff
@@ -18,11 +18,13 @@
 
 ColumnName = str
 
+TablesWithMaterializedColumns = Union[TableWithProperties, Literal["session_recording_events"]]
+
 TRIM_AND_EXTRACT_PROPERTY = "trim(BOTH '\"' FROM JSONExtractRaw(properties, %(property)s))"
 
 
 @cache_for(timedelta(minutes=15))
-def get_materialized_columns(table: TableWithProperties) -> Dict[PropertyName, ColumnName]:
+def get_materialized_columns(table: TablesWithMaterializedColumns) -> Dict[PropertyName, ColumnName]:
```

Comment:
This is missing a test.

Reply:
Ah good call - just added it.

Reply:
Hmmm this test is failing. I have to take off, but will look into later.